templates/zerver/accounts_home.html

```diff
@@ -78,7 +78,8 @@ <h1 class="get-started">{{ _("Sign up for Zulip") }}</h1>
                             <form class="form-inline" action="{{ backend.signup_url }}" method="get">
                                 <input type='hidden' name='multiuse_object_key' value='{{ multiuse_object_key }}' />
                                 <button id="register_{{ backend.button_id_suffix }}" class="login-social-button full-width"
-                                  {% if backend.display_icon %} style="background-image:url({{ backend.display_icon }})"  {% endif %}>
+                                  {% if backend.display_name == "Apple" %} style="background-image:url(https://appleid.cdn-apple.com/appleid/button/?width=328&height=50&color=white&locale={{ apple_locale }}&type=continue)"
+                              {% elif backend.display_icon %} style="background-image:url({{ backend.display_icon }})"  {% endif %}>
```

Comment:
I think we could still move this conditional to live in `zerver/views/portico.py` to edit the `display_icon` for the Apple backend for the two situations.